hazelcast-client-legacy/src/test/java/com/hazelcast/client/ClientRegressionWithMockNetworkTest.java

```diff
@@ -616,17 +615,23 @@ public void entryAdded(EntryEvent<Object, Object> event) {
     public void testDeadlock_WhenDoingOperationFromLifecycleListener() {
```

Comment:
Can we add the modified test as new test ? The old one with getting the partition table as on-demand also a valuable test to have. I think we need to have both versions at the same time.